torch/csrc/distributed/autograd/functions/recvrpc_backward.cpp

```diff
@@ -0,0 +1,33 @@
+#include <torch/csrc/distributed/autograd/functions/recvrpc_backward.h>
+#include <ATen/core/functional.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+using torch::autograd::Variable;
+
+torch::autograd::variable_list RecvRpcBackward::apply(
+    torch::autograd::variable_list&& grads) {
+  // Get size and options for tensors.
+  at::IntArrayRef sizes;
+  at::TensorOptions o;
+  for (const auto& v : grads) {
+    if (v.defined()) {
+      sizes = v.sizes();
+      o = static_cast<at::Tensor>(v).options();
```

Comment:
Does this mean, for now, we assume all tensors/grads are of the same sizes and type?